desktop/package/README.md

```diff
@@ -52,55 +58,62 @@ Haveno data folder on Mac: `/Users/<username>/Library/Application Support/Haveno
 
 Haveno data folder on Windows: `~\AppData\Roaming\Haveno\`
 
-## Copy installer and rebuild Haveno binaries
+## Copying installer and rebuilding Haveno binaries
 
 1. Copy the installer to a safe location because it will be deleted in the next step.
 2. `make clean && make` (or `make clean && make skip-tests`) to rebuild Haveno apps.
 
-
 ## Additional Notes
 
 ### Icons
 
-Icons (Haveno.zip) were obtained from https://github.com/haveno-dex/haveno-meta/issues/1#issuecomment-819741689.
+Icons (Haveno.zip) were obtained from <https://github.com/haveno-dex/haveno-meta/issues/1#issuecomment-819741689>.
 
-#### Linux
+### Building for Linux
 
 The linux package requires the correct packaging tools installed. You may run into the following errors:
 
-```
+```sh
 Error: Invalid or unsupported type: [deb]
 ```
-```
+
+```sh
 Error: Invalid or unsupported type: [rpm]
 ```
 
 On Ubuntu, resolve by running `sudo apt install rpm`. For deb, ensure dpkg is installed.
 
-```
+```sh
 Exception in thread "main" java.io.IOException: Failed to rename /tmp/Haveno-stripped15820156885694375398.tmp to /storage/src/haveno/desktop/build/libs/fatJar/desktop-1.0.0-SNAPSHOT-all.jar
-	at haveno.tools.Utils.renameFile(Utils.java:36)
-	at io.github.zlika.reproducible.StipZipFile.strip(StipZipFile.java:35)
-	at haveno.tools.DeterministicBuildTool.main(DeterministicBuildTool.java:24)
+ at haveno.tools.Utils.renameFile(Utils.java:36)
+ at io.github.zlika.reproducible.StipZipFile.strip(StipZipFile.java:35)
+ at haveno.tools.DeterministicBuildTool.main(DeterministicBuildTool.java:24)
 
 ```
 
 This may happen if the source folder is on a different hard drive than the system `tmp` folder. The tools-1.0.jar calls renameTo to rename the deterministic jar back to the fat jar location. You can temporarily change your temp directory on linux:
 
-```
+```sh
 export _JAVA_OPTIONS="-Djava.io.tmpdir=/storage/tmp"
 ```
 
-#### MacOs
+You will also need `flatpak` and `flatpak-builder`. On Debian/Ubuntu:
```

Comment:
These dependencies are always needed on Debian/Ubuntu?

In that case they should be added to the Linux instructions above, which currently has `sudo apt install -y rpm fuse`.